binders/kafka-binder/spring-cloud-stream-binder-kafka-core/src/main/java/org/springframework/cloud/stream/binder/kafka/provisioning/KafkaTopicProvisioner.java

```diff
@@ -542,6 +547,66 @@ private void alterTopicConfigsIfNecessary(AdminClient adminClient,
 		}
 	}
 
```

Comment:
KafkaMessageChannelBinder and ReactorKafkaBinder classes performed the same operations. I put these methods here to use them from a common place.